app/controllers/entry.rb

```diff
@@ -0,0 +1,73 @@
+include FileUtils::Verbose
+require 'fileutils'
+
+
+#create
+get '/upload' do
+  @cohorts    = Cohort.all
+  @languages  = Language.all
+  @phases     = Phase.all
+  erb :'entry/upload'
+end
+
+post '/upload' do
+  user_id     = current_user_id
+  phase_id    = Phase.find_by_name(params[:phase]).id
+  cohort_id   = Cohort.find_by_name(params[:cohort]).id
+  language_id = Language.find_by_name(params[:language]).id
+
+  if params[:file]
+    entry = Entry.create(params[:entry])
+    entry.update_attributes(phase_id:    phase_id, \
+                            cohort_id:   cohort_id, \
+                            user_id:     user_id \
+                           )
```

Comment:
You don't actually need the \ at the ends of the line to put this stuff on separate lines.